### ISSUE TITLE
NPC Note Moving to Top on Change

### ISSUE THREAD

**Issue by tmilktoast** [carlonicora/obsidian-rpg-manager#130]
I've been trying to save this report until I could figure out the circumstances of when and why it's happening. But no luck so far. 

On some of my NPCs, when I make any edits the note will jump back up to the top of the NonPlayerCharacter table. 

I don't believe it's a case of funky frontmatter. If I remove all of the links in a document it will stop doing it. Nothing shows up in the console when this happens. 

I tried remaking the same NPC from scratch using Create New NonPlayerCharacter, rather than using the Fill method which is how I made them originally, but the same issue persists. 

Here's frontmatter from one of them:

```
---
alias: [Cleopatra]
tags:
  - rpgm/npc/1
completed: false
synopsis: "an edgerunner and tech involved in the [[Dreamchip]] theft job. She installed the Cleopatra chip and has now taken over [[Jewel of the Nile Enterprises|JNE]], formerly HSE."
image: ""
goals: ""
pronoun: "s"
dates:
  dob: 12/23/2046
  death: 
---
```

**Comment by tmilktoast** [comment]
Thanks for getting back to me. I'll try to do that later tonight, which might be in 3-5 hours. I have to do parenty stuff for a while.

**Comment by tmilktoast** [comment]
Here's a screen capture. If I type a little, then stop, it goes right to the top (below the front matter). But it seems if I continue to type it sort of moves up and then back down. In this video I never scrolled up, only down. 

But, if you have big changes in store soon, maybe that'll have a side effect of fixing this.

I should clarify that this happens not only when I type, but when I make any change. For instance if I were to delete something. 

I also created a new NPC in a different vault that was basically empty and experienced the same behavior. In that vault i removed all other plugins and switched to the default theme.

https://user-images.githubusercontent.com/28547139/194178582-e1615abb-d799-4196-a389-aad977aa3446.mp4

**Comment by carlonicora** [comment]
@tmilktoast so, the timing seems do be consistent on when Obsidian saves your notes. When the note is saved, RPG Manager updates the metadata and refresh the views (headers, tables and so on). It should not jump up and down though, I will check it with the new version to make sure it does not happen.

Thanks for the screener though!!!

**Comment by carlonicora** [comment]
@tmilktoast this seems to be fixed in `v3.0`, I will ask you to test when I release next week

**Comment by tmilktoast** [comment]
> @tmilktoast this seems to be fixed in `v3.0`, I will ask you to test when I release next week

Thanks!
